crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -1,94 +1,83 @@
-use crate::Anchor;
-use bevy_asset::Handle;
+use bevy_asset::{Assets, Handle};
 use bevy_ecs::component::Component;
 use bevy_math::{Rect, Vec2};
 use bevy_reflect::{FromReflect, Reflect, TypeUuid};
-use bevy_render::{color::Color, texture::Image};
+use bevy_render::texture::Image;
 use bevy_utils::HashMap;
 
-/// An atlas containing multiple textures (like a spritesheet or a tilemap).
+/// Maps a layout for a texture. Used with the [`TextureAtlas`] component it allows to
+/// either draw a specific area of the target texture, or to animate a sprite sheet.
```

Comment:
```suggestion
/// Stores a map used to lookup the position of a texture in a [`TextureAtlas`].
/// This can be used to either use and look up a specific section of a texture, or animate frame-by-frame as a sprite sheet.
```